libecole/src/scip/row.cpp

```diff
@@ -33,7 +33,8 @@ int RowProxy::n_lp_nonz() const noexcept {
 }
 
 real RowProxy::l2_norm() const noexcept {
-	return SCIProwGetNorm(value);
+	auto norm = SCIProwGetNorm(value);
+	return norm > 0 ? norm : 1.;
```

Comment:
I do not feel this belongs here, as it does not match the name of the function.
Should go in `NodeBipartite`.
https://github.com/ds4dm/ecole/blob/3d6f3405223f93193b72b2acc278ea4f009a70a5/libecole/src/observation/nodebipartite.cpp#L35
Should go https://github.com/ds4dm/ecole/blob/3d6f3405223f93193b72b2acc278ea4f009a70a5/libecole/src/observation/nodebipartite.cpp#L84

Reply:
I changed the code according to your recommendations.